R/utilities-population.R

```diff
@@ -0,0 +1,8 @@
+#' @export
+loadPopulation <- function(fileName, simulation) {
+  validateIsOfType(simulation, "Simulation")
+  validateIsString(fileName)
+  populationImporter <- getNetTask("PopulationImporter")
+  population <- rClr::clrCall(populationImporter, "ImportPopulation", fileName, simulation$ref)

```

Comment:
This is in fact not really a population but a `bag` of parmaeter values by path. Because I do not think we will have a Population object in R, I think it makes sense to call this `bag` a population unless you see a problem